napi_module_registration.md

```diff
@@ -0,0 +1,76 @@
+# Module registration
+
+N-API based modules are registered in the same manner as other modules
+except that instead of using the `NODE_MODULE` macro you use:
+
+```C
+NAPI_MODULE(addon, Init)
+```
+
+The next difference is the signature for the `Init` method.  For a N-API
+based module it is as follows:
+
+```C
+void Init(napi_env env, napi_value exports, napi_value module, void* priv);
+```
+
+As with any other module, functions are exported by either adding them to
+the exports or module objects passed to the Init method.
```

Comment:
&#96;exports&#96; or &#96;module&#96; objects passed to the &#96;Init&#96; method.